addons/event_sale/views/event_views.xml

```diff
@@ -88,6 +88,20 @@
             <field name="model">event.event</field>
             <field name="inherit_id" ref="event.view_event_form"/>
             <field name="arch" type="xml">
+                <xpath expr="//div[hasclass('oe_button_box')]" position="inside">
+                    <field name="currency_id" invisible="1"/>
+                    <button name="sale_order_action_view_from_event"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-dollar"
+                            help="Total sales for this event"
+                            attrs="{'invisible': ['|', ('sale_total_price', '=', 0), ('sale_total_price', '=', False)]}">
+                        <span class="mh-100">
```

Comment:
Remove mh-100.

The display is arguably better with it but it's not done for any of the other buttons so let's not bother.